src/lib/checkVersion.ts

```diff
@@ -77,6 +78,7 @@ export class CheckVersion {
   getVersion(manager: ManagerType, location: string) {
     if (manager === 'cargo') return this.handleCargo(location)
     if (manager === 'npm') return this.handlePackageJson(location)
+    if (manager === 'poetry') return this.handlePoetry(location)
 
     throw new Error(`unknown manager type - [${manager}]`)
```

Comment:
could change this to use `switch(manager)`  with the `default` throwing the error, rather than having stacked ifs.